test/extended/util/framework.go

```diff
@@ -275,6 +286,17 @@ func ExtendedTestPath() string {
 	return os.Getenv("EXTENDED_TEST_PATH")
 }
 
+//ArtifactDirPath returns the value of ARTIFACT_DIR environment variable
+func ArtifactDirPath() string {
+	return os.Getenv("ARTIFACT_DIR")
+}
```

Comment:
ok ... will do.  Do you want to include that change for the pre-existing EXTENDED_TEST_PATH in addition to the ARTIFACT_DIR I added ?